applications/app/controllers/ShortUrlsController.scala

```diff
@@ -19,7 +19,7 @@ object ShortUrlsController extends Controller with Logging with ExecutionContext
       response.content.map(_.id).map { id =>
         Redirect(LinkTo(s"/$id"), queryString)
       }.getOrElse(NotFound)
-    }.recover(convertApiExceptionsWithoutEither).map(Cached(60))
+    }.recover(convertApiExceptionsWithoutEither).map(Cached.explicitlyCache(1800))
```

Comment:
we could soft purge those, if needed, shouldn't happen too often. Alternatively, we can give fresher content a smaller cache time.